examples/routing_service/routing_service_msql_adapter/cpp/rs_dll/CMakeLists.txt

```diff
@@ -0,0 +1,89 @@
+#
+# (c) 2018 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+#
+#  RTI grants Licensee a license to use, modify, compile, and create derivative
+#  works of the Software.  Licensee has the right to distribute object form
+#  only for use with RTI products.  The Software is provided "as is", with no
+#  warranty of any type, including any warranty for fitness for any purpose.
+#  RTI is under no obligation to maintain or support the Software.  RTI shall
+#  not be liable for any incidental or consequential damages arising out of the
+#  use or inability to use the software.
+
+
+cmake_minimum_required(VERSION 3.12)
```

Comment:
We are trying to use the minimal CMake version needed by the different examples. For the most of them, that version is CMake 3.7. Since you need CMake 3.12, I am not sure what would be the best approach:
* Increase the required CMake version in all the examples
* Skip this example if the version is lower than 3.12

@fgaranda, what do you think?

Reply:
@iblancasa The later sounds better. How come we need CMake 3.12?

Reply:
The other routing service examples use 3.11. I will change it to that. I don't think I need 3.12

Reply:
Changed to 3.11. Will commit all changes at once.

Reply:
You need CMake 3.12 because it is the first version where FindODBC is available:
https://cmake.org/cmake/help/v3.12/module/FindODBC.html